patches/minecraft/net/minecraft/entity/player/PlayerEntity.java.patch

```diff
@@ -336,5 +348,9 @@
 +         else if (facing.func_176740_k().func_176722_c()) return playerEquipmentHandler.cast();
 +      }
 +      return super.getCapability(capability, facing);
++   }
++
++   public void setForcedPose(@Nullable Pose pose) {
```

Comment:
Why is this being forced at all? The data manager already syncs the pose, why add some backdoor to bypass it?

Reply:
Unfortunately, the player entity (re)determines the players pose every tick and thereby overrides any manually set pose.
To determine the current pose the PlayerEntity uses hardcoded sizes and therefore e.g. falls back to the swimming pose when the player is in a 1x1 space (where they fit because of actual entity size). 
It is possible to override the value again right after that in the Post PlayerTickEvent, so the pose determined by vanilla does not affect rendering or game mechanics. However, every time the pose is changed `notifyDataManagerChange` calls `recalculateSize` causing an event to be posted.
Therefore, without this "backdoor" the event would be posted twice every tick instead of not at all.

If you still prefer this, I can remove this patch.